string.c

```diff
@@ -433,6 +433,10 @@ fstr_update_callback(st_data_t *key, st_data_t *value, st_data_t data, int exist
 
         if (rb_objspace_garbage_object_p(str)) {
             arg->fstr = Qundef;
+            // When RSTRING_FSTR strings are sweeped, they call `st_delete`.
```

Comment:
Grammar nit:

```suggestion
            // When RSTRING_FSTR strings are swept, they call `st_delete`.
```

Reply:
Thanks, I probably should code in Grammarly.